src/Microsoft.ML.Data/Evaluators/Metrics/AnomalyDetectionMetrics.cs

```diff
@@ -20,7 +20,7 @@ public sealed class AnomalyDetectionMetrics
         /// a randomly chosen positive instance higher than a randomly chosen negative one
         /// (assuming 'positive' ranks higher than 'negative').
         /// </remarks>
-        public double Auc { get; }
+        public double AreaUnderTheCurve { get; }
```

Comment:
>AreaUnderTheCurve [](start = 22, length = 17)

This comment addresses all variants of AUC in the various metrics classes.

Technically this name should be `AreaUnderTheRocCurve` if we want to really be expressive. Area under the curve is a general thing that you want to do with a collection of points. I know that sci-kit learn actually ended up moving the ROC-AUC calculation into the `roc_auc_score` method and made `auc` a generic method, so that's a sign that plain AUC might be misleading.

And don't forget, people may ask for the AUPRC soon, so we may want to make the change now so that it's less confusing later.